devdocs/settingUp.md

```diff
@@ -209,6 +210,8 @@ TBD
     deployed app and not the dev server).
     Note that GAE daily quota will be exhausted after 2-3 runs of the full test suite.
 
+4. (Optional) After installing Google Cloud SDK and deploying to your staging server, run `gcloud auth login` in the terminal of your PC and choose your google account for authentication.
```

Comment:
These are not 'optional' steps, but additional steps needed for a specific task. Add a separate section (e.g. Running client scripts) in an appropriate place in the doc, and explain what client scripts are and what extra set up steps are required. Also mention that most developers don't need to do it.